platforms/documentation/docs/src/docs/userguide/optimizing-performance/isolated_projects.adoc

```diff
@@ -16,44 +16,45 @@
 
 = Isolated Projects
 
-Isolated Projects is a pre-alpha Gradle feature that extends the configuration cache to further improve performance, in particular the performance of Android Studio and IDEA sync.
+_Isolated Projects_ is a pre-alpha Gradle feature that extends the configuration cache to further improve performance, particularly the performance of Android Studio and IDEA sync.
 
-When isolated projects is enabled, the configuration model of Gradle projects are "isolated" from each other.
+When _isolated projects_ is enabled, the configuration model of Gradle projects are "isolated" from each other.
```

Comment:
❓ Should we always capitalize feature names anywhere in a sentence, i.e., _Isolated Projects_ instead of _isolated projects_? My preference would be to always consistently capitalize them so clarity is not predicated upon a particular choice of formatting.

Reply:
It's common to capitalize "product" name. In this case, I would consider IP a product and capitalize. I'll update accordingly and merge.